mypyc/test-data/run-misc.test

```diff
@@ -875,6 +875,56 @@ assert call_all(mixed_110) == 1
 assert call_any_nested([[1, 1, 1], [1, 1], []]) == 1
 assert call_any_nested([[1, 1, 1], [0, 1], []]) == 0
 
+
+[case testSum]
+from typing import Any, List
```

Comment:
Adding `[typing fixtures/typing-full.pyi]` here is the simplest way to solve test errors.